robottelo/constants/__init__.py

```diff
@@ -2088,6 +2106,10 @@
     "content_export_library",
     "content_export_repository",
     "content_export_version",
+    "content_import_info",
+    "content_import_library",
+    "content_import_repository",
+    "content_import_version",
```

Comment:
these tests are new and failing -- we can add them now in failing state or wait and add them with corrected setup

Reply:
I§d prefer to add them with corrected steps. @lpramuk @rmynar what you think?

Reply:
I have nothing against adding new steps in failing state if we track a task to fix them

Reply:
I think I'll split this out, and also add a check that we really have all modules covered: #17840